src/combinatorics/simple_graph/subgraph.lean

```diff
@@ -262,9 +262,30 @@ instance : bounded_order (subgraph G) :=
 
 @[simp] lemma not_bot_adj {v w : V} : ¬(⊥ : subgraph G).adj v w := not_false
 
+@[simp] lemma inf_adj {H₁ H₂ : subgraph G} {v w : V} :
+  (H₁ ⊓ H₂).adj v w ↔ H₁.adj v w ∧ H₂.adj v w := iff.rfl
+
+@[simp] lemma sup_adj {H₁ H₂ : subgraph G} {v w : V} :
+  (H₁ ⊔ H₂).adj v w ↔ H₁.adj v w ∨ H₂.adj v w := iff.rfl
+
+@[simp] lemma edge_set_top : (⊤ : subgraph G).edge_set = G.edge_set :=
+set.ext (λ e, sym2.ind (by simp) e)
+
+@[simp] lemma edge_set_bot : (⊥ : subgraph G).edge_set = ∅ :=
+set.ext (λ e, sym2.ind (by simp) e)
+
+@[simp] lemma edge_set_inf {H₁ H₂ : subgraph G} : (H₁ ⊓ H₂).edge_set = H₁.edge_set ∩ H₂.edge_set :=
+set.ext (λ e, sym2.ind (by simp) e)
+
+@[simp] lemma edge_set_sup {H₁ H₂ : subgraph G} : (H₁ ⊔ H₂).edge_set = H₁.edge_set ∪ H₂.edge_set :=
+set.ext (λ e, sym2.ind (by simp) e)
+
 @[simp] lemma spanning_coe_top : (⊤ : subgraph G).spanning_coe = G :=
 by { ext, refl }
 
+@[simp] lemma spanning_coe_bot : (⊥ : subgraph G).spanning_coe = ⊥ :=
+by { ext, refl }
```

Comment:
Apparently this is true by `rfl`.
```suggestion
rfl
```

By the way, we probably should change the `subgraph.bot` definition to
```
def bot : subgraph G :=
{ verts := ∅,
  adj := ⊥,
  adj_sub := λ v w h, false.rec _ h,
  edge_vert := λ v w h, false.rec _ h,
  symm := λ u v h, h }
```
This uses `⊥` for `adj`

Reply:
Amended, thanks